Cargo.lock

```diff
@@ -469,7 +469,7 @@ checksum = "8c3c1a368f70d6cf7302d78f8f7093da241fb8e8807c05cc9e51a125895a6d5b"
 
 [[package]]
 name = "benchmarks"
-version = "1.3.2"
```

Comment:
@Kerollmops I don't know why we have this git diff knowing the version in the `release-v1.4.0` branch is already `v1.4.0`...

Reply:
I really don't know, maybe the first commit of this PR, but who knows, git is a mystery 🤷‍♂️